CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_hubspot v0.17.2
+[PR #142](https://github.com/fivetran/dbt_hubspot/pull/142) includes the following updates:
+
+## 🪲 Bug Fixes 🪛
+- Fixed the `fivetran_utils.enabled_vars` conditional by adding the `hubspot_contact_list_member_enabled` variable in `hubspot_contact_lists` to solve for compilation errors when the `contact_list` source table is not being used. If `hubspot_contact_list_member_enabled` is `true`, `int_hubspot__email_metrics__by_contact_list` is now brought in as a dependency, and ignored otherwise. 
```

Comment:
A few suggested changes to this CHANGELOG entry. 

```suggestion
- Fixed the `fivetran_utils.enabled_vars` conditional by adding the `hubspot_contact_list_member_enabled` variable in the `hubspot__contact_lists` model. This solves for compilation errors when the `contact_list` source table is not synced in the destination. If `hubspot_contact_list_member_enabled` is `true`, `int_hubspot__email_metrics__by_contact_list` is brought in as a dependency, and ignored otherwise. 
```